### PR TITLE
Add R language support

### PR DESCRIPTION
**Pull request by Novemser**
Add a shabby thin layer to support `TiSpark` in R language.

### CODE REVIEW

File: TiSparkR/R/tisparkR.R [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Title     : TiSparkR
```
Comment (ilovesoup) [inline TiSparkR/R/tisparkR.R]: Please add copyright info

File: TiSparkR/Readme.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+## TiSparkR
```
Comment (ilovesoup) [inline TiSparkR/Readme.md]: Looks like merging to root/README.md is better

File: TiSparkR/R/tisparkR.R [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+# Objective : TiSpark entry for R
+# Created by: novemser
+# Created on: 17-11-1
+
+createTiContext <- function(session) {
```
Comment (ilovesoup) [inline TiSparkR/R/tisparkR.R]: how error handling behaves here in R?

File: TiSparkR/DESCRIPTION [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Package: TiSparkR
+Type: Package
+Title: TiSpark for R
+Version: 0.1.0
+Author: Novemser
+Maintainer: The package maintainer <novemser@gmail.com>
+Description: Add a shabby thin layer to support TiSpark in R language.
+License: Apache 2.0
+Encoding: UTF-8
+LazyData: true
```
Comment (ilovesoup) [inline TiSparkR/DESCRIPTION]: what is lazydata?

Comment (Novemser) [reply TiSparkR/DESCRIPTION]: If the `DESCRIPTION` contains `LazyData: true`, then datasets in R will be lazily loaded. This means that they won’t occupy any memory until we use them. It is recommended that we always include `LazyData: true` in the DESCRIPTION.

File: TiSparkR/R/tisparkR.R (hunk 1 above)
Comment (Novemser) [reply TiSparkR/R/tisparkR.R]: Done.

**Pull request merged**